packages/components/src/EditableText/EditableText.component.js

```diff
@@ -37,17 +38,18 @@ export function PlainTextTitle({ onEdit, disabled, text, inProgress, feature, t
 		</div>
 	);
 }
-
 PlainTextTitle.propTypes = {
-	text: PropTypes.string.isRequired,
-	onEdit: PropTypes.func.isRequired,
+	componentClass: PropTypes.string,
```

Comment:
that means I can t use a componetn but only native. PLease add oneOf(string, func)